sql/logic_test.go

```diff
@@ -424,13 +437,43 @@ func (t *logicTest) execQuery(query logicQuery) {
 	}
 }
 
+func (t *logicTest) success(file string) {
+	t.progress++
+	now := time.Now()
+	if now.Sub(t.lastProgress) >= 2*time.Second {
+		t.lastProgress = now
+		fmt.Printf("%s: %d\n", file, t.progress)
+	}
+}
+
 func TestLogic(t *testing.T) {
 	defer leaktest.AfterTest(t)
 
-	paths, err := filepath.Glob(*testdata)
-	if err != nil {
-		t.Fatal(err)
+	var globs []string
+	if *bigtest {
+		const logicTestPath = "../../sqllogictest"
+		if _, err := os.Stat(logicTestPath); os.IsNotExist(err) {
+			t.Fatalf("unable to find sqllogictest repo: %s\n"+
+				"git clone https://github.com/cockroachdb/sqllogictest ../../sqllogictest",
```

Comment:
this should print an absolute path because i sometimes use `make -c` :smiling_imp: